lib/util/util.c

```diff
@@ -1504,7 +1507,10 @@ bool util_sscanf_double(const char * buffer , double * value) {
    Base 8
 */
 
-bool util_sscanf_octal_int(const char * buffer , unsigned int * value) {
+bool util_sscanf_octal_int(const char * buffer , int * value) {
```

Comment:
Well - I would prefer if it were unchanged, the only current use for it is for reading modes.